src/AFQMC/Estimators/FullObsHandler.hpp

```diff
@@ -111,20 +111,20 @@ class FullObsHandler : public AFQMCInfo
       std::transform(cname.begin(), cname.end(), cname.begin(), (int (*)(int))tolower);
       if (cname == "onerdm")
       {
-        properties.emplace_back(Observable(std::move(full1rdm(TG, info, cur, walker_type, nave, block_size))));
```

Comment:
In this case, temporal full1rdm type object is created and then calling move constructor.

Reply:
In addition, if std::move is needed on the full1rdm. std::move is needed on Observable as well.